src/misclibs.F

```diff
@@ -4583,21 +4370,14 @@ subroutine tmf(nstat,rstat,ruh,rvh,rho,wa)
         foo1(k) = foo1t
         foo2(k) = foo2t
       enddo
-
       tmfu=0.0d0
       tmfd=0.0d0
-#ifndef _B4B10R
       !$acc loop vector reduction(+:tmfu,tmfd)
-#endif
       do k=1,nk
         tmfu=tmfu+foo1(k)
         tmfd=tmfd+foo2(k)
       enddo
```

Comment:
Thanks for reporting the internal compiler error for `tmfu` and `tmfd`. See my previous comment about separating the kernels here.